packages/mobile/src/components/Feed/MainContent/SocialFeed.tsx

```diff
@@ -69,8 +69,9 @@ const SocialFeedMainContent = memo(() => {
     );
 
     const postContent = post?.fileMetadata.appData.content;
+    console.log('postContent', postContent);
     navigation.navigate('Post', {
-      odinId: post?.fileMetadata.senderOdinId,
+      odinId: post?.fileMetadata.originalAuthor,
```

Comment:
This can't be right I'm afraid, the senderOdinId has to be location where the post is stored; The originalAuthor can be anyone with write access on that drive. But when looking it up, you'll need to find the place where it's stored, not who authored it